package.json

```diff
@@ -1,42 +1,38 @@
 {
   "name": "tailor",
   "version": "3.2.2",
-  "codename": "silk",
+  "private": true,
```

Comment:
Hmm, while I don't mind rearranging stuff inside `package.json` that has nothing to do with `vue-cli` introduction...